test_scripts/Smoke/API/028_AlertManeuver_PositiveCase.lua

```diff
@@ -0,0 +1,167 @@
+--
+-- Expected:
+-- SDL validates parameters of the request
+-- SDL checks if Navigation interface is available on HMI
+-- SDL checks if TTS interface is available on HMI
+-- SDL checks if AlertManeuver is allowed by Policies
+-- SDL checks if all parameters are allowed by Policies
+-- SDL transfers the Navigation part of request with allowed parameters to HMI
+-- SDL transfers the TTS part of request with allowed parameters to HMI
+-- SDL receives Navigation part of response from HMI with "SUCCESS" result code
+-- SDL receives TTS part of response from HMI with "SUCCESS" result code
+-- SDL transfers response to mobile app
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSmoke = require('test_scripts/Smoke/commonSmoke')
+local commonFunctions = require("user_modules/shared_testcases/commonFunctions")
+
+--[[ Local Variables ]]
+local putFileParams = {
+  requestParams = {
+    syncFileName = 'icon.png',
+    fileType = "GRAPHIC_PNG",
+    persistentFile = false,
+    systemFile = false
+  },
+  filePath = "files/icon.png"
+}
+
+local requestParams = {
+  ttsChunks = {
+    {
+      text = "FirstAlert",
+      type = "TEXT",
+    },
+    {
+      text = "SecondAlert",
+      type = "TEXT",
+    },
+  },
+  softButtons = {
+    {
+      type = "BOTH",
+      text = "Close",
+      image = {
+        value = "icon.png",
+        imageType = "DYNAMIC",
+      },
+      isHighlighted = true,
+      softButtonID = 821,
+      systemAction = "DEFAULT_ACTION",
+    },
+    {
+      type = "BOTH",
+      text = "AnotherClose",
+      image = {
+        value = "icon.png",
+        imageType = "DYNAMIC",
+      },
+      isHighlighted = false,
+      softButtonID = 822,
+      systemAction = "DEFAULT_ACTION",
+    },
+  }
+}
+
+local responseNaviParams = {}
+responseNaviParams.softButtons = commonFunctions:cloneTable(requestParams.softButtons)
+for k, _ in pairs(responseNaviParams.softButtons) do
+  if responseNaviParams.softButtons[k].image then
+    responseNaviParams.softButtons[k].image.value = commonSmoke.getPathToFileInStorage(responseNaviParams.softButtons[k].image.value)
+  end
+end
+
+local responseTtsParams = {}
+responseTtsParams.ttsChunks = requestParams.ttsChunks
+
+local allParams = {
+  requestParams = requestParams,
+  responseNaviParams = responseNaviParams,
+  responseTtsParams = responseTtsParams
+}
+
+--[[ Local Functions ]]
+local function PTUpdateFunc(tbl)
+  local AlertMgroup = {
+    rpcs = {
+      AlertManeuver = {
+        hmi_levels = { "BACKGROUND", "FULL", "LIMITED" },
+      }
+    }
+  }
+  tbl.policy_table.functional_groupings.NewTestCaseGroup = AlertMgroup
+  tbl.policy_table.app_policies[config.application1.registerAppInterfaceParams.appID].groups =
+  { "Base-4", "NewTestCaseGroup" }
+end
+
+local function ExpectOnHMIStatusWithAudioStateChanged_AlertManeuver(self)
```

Comment:
@GetmanetsIrina as I could see this function is already implemented in `commonSmoke.expectOnHMIStatusWithAudioStateChanged`. Is it possible to reuse it?

Reply:
@AKalinich-Luxoft , expectOnHMIStatusWithAudioStateChanged function is for Alert RPC, is not common function, so can not be reused in case with AlertManeuver.